Presentation.Web/app/components/it-system/it-system-overview.controller.ts

```diff
@@ -812,7 +812,16 @@
                         .withId("note")
                         .withFilteringOperation(Utility.KendoGrid.KendoGridColumnFiltering.Contains)
                         .withContentOverflow()
-                        .withSourceValueEchoRendering());
+                        .withSourceValueEchoRendering())
+                .withColumn(builder =>
+                    builder
+                        .withDataSourceName("RiskAssessmentDate")
+                        .withTitle("Risk assessment date <- change the text")
```

Comment:
change text

Reply:
why not use the UI text 
![image](https://user-images.githubusercontent.com/13116889/206158400-657689c6-7d15-4a2f-bbfa-f195b6ea0598.png)